src/Order.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace Razorpay\Api;
+
+class Order extends Entity
+{
+    /**
+     *  @param $id Order id description 
+     */
+    public function payments($id)
```

Comment:
This is breaking our existing convention of api->entity->fetch(id)->subentity and shifting to api->entity->subentity(id). Just pointing out @shashankkumar